config/phpunit.lando.xml

```diff
@@ -23,13 +23,13 @@
     <!-- Do not limit the amount of memory tests take to run. -->
     <ini name="memory_limit" value="-1"/>
     <!-- Example SIMPLETEST_BASE_URL value: http://localhost -->
-    <env name="SIMPLETEST_BASE_URL" value=""/>
+    <env name="SIMPLETEST_BASE_URL" value="http://nidirect-d8.lndo.site"/>
```

Comment:
I'd look at dumping in the lando site url env variable as we have different lando configs/names. 
We should also add in the generic database parameters that the d8 recipe provides.